release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,10 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 25 April 2024 - Enhancement - Catalog - Show DMS issues when deploying a catalog item [ID_39374]
+
+Preventing users from deploying a catalog item when a deployment will fail. Showing the different issues with a documentation link for each DataMiner System.
```

Comment:
OK to rephrase this like this? 
```suggestion
In case deploying a catalog item to a DataMiner System will fail, it will now no longer be possible for users to try to deploy the item to that system, and a documentation link will be shown so the users can resolve the issue.```

Reply:
This looks good to me.